packages/headless/src/features/commerce/facets/facet-set/interfaces/response.ts

```diff
@@ -11,7 +11,16 @@ export type CategoryFacetResponse = BaseFacetResponse<
 > &
   CategoryFacetDelimitingCharacter;
 
-export interface CategoryFacetValue extends BaseFacetValue {
+interface NonLocationFacetValue {
```

Comment:
Two things:
- Location facet values **do not** have a `numberOfResults`. I'll let @pvanool or @BenKandelaars expand on this!
- Only category facet values **do** have `moreValuesAvailable`. It looks like our facet valus are mistyped in headless... because Commerce API also exposes `moreValuesAvailable` on non-category facet values! I'll mark the `moreValuesAvailable` fields on non-category facet values as deprecated. I'll also adjust Commerce API to reflect this.

Reply:
Aah yes indeed it makes sense that category facet values are the only type of values that would need the `moreValuesAvailable` property; other facet value types can't be expanded.

I'm curious about not supporting `numberOfResults` on location facets; it seems like an important feature to me.

Reply:
It seemed confusing to the end user. Other facets on products give the number of products. This facet is on availability document (store). And generally there is only one store. The team decided to remove it.